src/room/InCallView.tsx

```diff
@@ -222,7 +223,9 @@ export const InCallView: FC<InCallViewProps> = ({
     room: livekitRoom,
   });
 
-  const [muteAllAudio] = useSetting(muteAllAudioSetting);
+  const muteAllAudioControlled = useObservableEagerState(setOutputEnabled$);
+  const [muteAllAudioFromSetting] = useSetting(muteAllAudioSetting);
+  const muteAllAudio = muteAllAudioControlled || muteAllAudioFromSetting;
```

Comment:
I would expect this `setOutputEnabled$` subject to be consumed by the CallViewModel, piped through `startWith` to give it an initial value, combined with the setting there, and exposed as a single state observable. (I'm actually surprised if this isn't crashing the InCallView component currently because `setOutputEnabled$`, being a subject, doesn't provide an initial value.)

Tangentially, I think you forgot a `!`:

```suggestion
  const muteAllAudioControlled = !useObservableEagerState(setOutputEnabled$);
  const [muteAllAudioFromSetting] = useSetting(muteAllAudioSetting);
  const muteAllAudio = muteAllAudioControlled || muteAllAudioFromSetting;
```

Reply:
I am not sure this is possible since we need the mute info also in the groupCallView

Reply:
moved it into its own file and fixed the `!` mistake.